enterprise/backend/test/metabase_enterprise/advanced_permissions/driver/impersonation_test.clj

```diff
@@ -97,6 +97,42 @@
                          mt/process-query
                          mt/rows))))))))))
 
+(deftest conn-impersonation-test-redshift
```

Comment:
These individual driver tests are in the generic `metabase-enterprise.advanced-permissions.driver.impersonation-test` because they need to be in an EE namespace to use the `with-impersonations` macro, and we don't have driver-specific EE test namespaces already.

If we add a lot more drivers we can rethink the organization but I'm sticking this one here too for the time being.